Format

alexMercy opened the pull request:
If Closes #6

alexMercy committed:
refactor controller template

alexMercy committed:
add format script and apply prettier config

alexMercy committed:
format with 120 line length

pavlumik reviewed (approved):
я бы мержил в последнюю очередь... конфликты решать будет явно не просто..


--ans: Я дальше работал на основе format branch просто

alexMercy committed:
Resolve conflicts

Merge branch 'main' of https://github.com/alexMercy/nest9-swagger-codegen into format
